File: letter-box/src/main/java/com/murali/letterbox/service/MovieService.java
package com.murali.letterbox.service;

import java.util.List;

import com.murali.letterbox.model.Movie;

public interface MovieService {
	List<Movie> getAllMovies();
	String addMovie(Movie movie) throws Exception;
	Movie getMovie(String movie) throws Exception;
	Movie updateMovie(String name, Movie movie) throws Exception;
	Movie deleteMovie(String name) throws Exception;
}

File: reactive-letter-box/src/services/MovieService.js
import axios from "axios";

const MOVIES_API_BASE_URL = "http://localhost:8080/letterbox/movies/";
class EmployeeService{
    getAllMovies(){
        return axios.get(MOVIES_API_BASE_URL);
    }
    getMovie(name){
        return axios.get(MOVIES_API_BASE_URL+name);
    }
}

export default new EmployeeService();